From c4cfe37d8c72a3469e6f71140a8e6ce81857af41 Mon Sep 17 00:00:00 2001
From: Aleksander Machniak <machniak@kolabsys.com>
Date: Mon, 22 May 2017 13:03:24 +0200
Subject: [PATCH] Fix regression where declining an event from Calendar would
 send CANCEL instead of REPLY (Bifrost#T31474)

The fix for T1357 (acc49b51fff21) was broken for cases with delegation and events in other user namespace.
---
 plugins/calendar/calendar_ui.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/calendar/calendar_ui.js b/plugins/calendar/calendar_ui.js
index edbc60e..cf9a4d4 100644
--- a/plugins/calendar/calendar_ui.js
+++ b/plugins/calendar/calendar_ui.js
@@ -2562,7 +2562,7 @@ function rcube_calendar_ui(settings)
       if (_has_attendees) {
         var checked = (settings.itip_notify & 1 ? ' checked="checked"' : '');
 
-        if (action == 'remove' && cal.group == 'personal' && is_attendee(event)) {
+        if (action == 'remove' && cal.group != 'shared' && is_attendee(event)) {
           decline = true;
           checked = event.status != 'CANCELLED' ? checked : '';
           html += '<div class="message">' +
-- 
2.9.4

